python/ray/node.py

```diff
@@ -683,3 +734,110 @@ def remaining_processes_alive(self):
             True if any process that wasn't explicitly killed is still alive.
         """
         return not any(self.dead_processes())
+
+
+class LocalNode(object):
+    """Imitate the node that manages the processes in local mode."""
+
+    @property
+    def node_ip_address(self):
+        """Get the cluster Redis address."""
+        logger.warning("Local node does not have an IP address.")
+        return None
+
+    @property
+    def redis_address(self):
+        """Get the cluster Redis address."""
+        logger.warning("Local node does not have a Redis address.")
+        return None
+
+    @property
+    def redis_password(self):
+        """Get the cluster Redis password"""
+        logger.warning("Local node does not have a Redis password.")
+        return None
```

Comment:
I can't see why we return `None` in many properties. 

Does `LocalNode` extends from `Node`? If yes, let's say `LocalNode(Node)`.
If just want to declare the same interfaces as Node's, let's throw exceptions instead of many `logging.warning`.